model/src/main/proto/exploration.proto

```diff
@@ -85,7 +85,50 @@ message Interaction {
   Outcome default_outcome = 6;
   // Mapping from the name of a customization arg to the default interaction
   // object of the customization arg
-  map<string, InteractionObject> customization_args = 7;
+  map<string, SchemaObject> customization_args = 7;
+}
+
+// Structure for schema objects. See https://github.com/oppia/oppia/blob/fcb44f8/schema_utils.py#L50 for context.
+message SchemaObject {
+  oneof object_type {
+    bool bool_value = 1;
+    CustomSchemaValue custom_schema_value = 2;
+    DictObject dict_value = 3;
+    float float_value = 4;
```

Comment:
Unused--please remove.

Reply:
Removed this for now but might need to introduce this back in future keeping https://github.com/oppia/oppia/blob/fcb44f8/schema_utils.py#L50 in mind.

Resolving this on behalf of Ben.